Build: Unify custom dockerfiles, fix missing semicolon (#59615)
grafana/grafana: docs/sources/setup-grafana/installation/docker/index.md

You can build your own customized image that includes plugins. This saves time if you are creating multiple images and you want them all to have the same plugins installed on build.

In the [Grafana GitHub repository](https://github.com/grafana/grafana) there is a folder called `packaging/docker/custom/`, which includes a Dockerfile that can be used to build a custom Grafana image. It accepts `GRAFANA_VERSION`, `GF_INSTALL_PLUGINS`, and `GF_INSTALL_IMAGE_RENDERER_PLUGIN` as build arguments.

> The `GRAFANA_VERSION` build argument needs to be a valid `grafana/grafana` docker image tag. By default this will build an Alpine-based image. To build an Ubuntu-based image, append `-ubuntu` to the `GRAFANA_VERSION` build argument (Grafana v6.5+).

Reviewer: ```suggestion
The `GRAFANA_VERSION` build argument must be a valid `grafana/grafana` docker image tag. By default, this builds an Alpine-based image. To build an Ubuntu-based image, append `-ubuntu` to the `GRAFANA_VERSION` build argument (available in Grafana v6.5 and later).
```

> Example of how to build and run:

Reviewer: Looking for a bit more explanation here about the following command.

Is it: "The following example shows you how to build and run a Grafana image in Docker."? Or something similar?

Author: We could say something like this:
```suggestion
The following example shows you how to build and run a custom Grafana docker image based on the latest official ubuntu-based Grafana docker image:
```

Author: @chri2547 thoughts on ^?

Reviewer: Looks fine, just capitalize the two instances of Docker and capitalize Ubuntu.
